src/jaro-winkler.ts

```diff
@@ -72,13 +72,21 @@ export function distance(s1: string, s2: string) {
 export type JaroWinklerOptions = {
     ignoreCase?: boolean;
     similarityThreshold?: number;
+    language?: string;
+    ignoreLocalization?: boolean;
 };
```

Comment:
**suggestion:** We should add docstrings to these too

Reply:
Yeah, I'll wait to see if the langCodes are added